collect_app/src/main/AndroidManifest.xml

```diff
@@ -288,7 +288,6 @@ the specific language governing permissions and limitations under the License.
         Do not use the activity directly as its name/package might change. -->
         <activity
             android:name=".external.FormUriActivity"
-            android:noHistory="true"
```

Comment:
As far as I can see we aren't using this any longer now we explicitly call `finish()` in all cases. `noHistory` doesn't work with activity results as the calling Activity is finished as soon as the user hits the next Activity.